vue/src/views/Edit.vue

```diff
@@ -1,5 +1,98 @@
 <template>
-  <div class="edit">
-    <h1>This is an Edit page</h1>
+  <div>
+    <div>
+      <Header />
+    </div>
+    <div class="formOut">
+      <!-- formではなくbindで書き換えてJSに渡す -->
+      <ul>
+        <li>
+          <label class="tag" for="editAge">年齢</label>
+          <input id="editAge" type="number" name="editAge">
+        </li>
+        <li>
+          <label class="tag" for="editScore">スコア</label>
+          <input id="editScore" type="number" name="editScore">
+        </li>
+        <li>
+          <label class="tag" for="editComment">コメント</label>
+          <input id="editComment" type="text" name="editComment">
+        </li>
+      </ul>
+      <button class="graphRegister" type="submit">
+        登録
+      </button>
+      <button class="graphEdit" type="submit">
+        編集
+      </button>
+    </div>
+    <div class="chart">
+      <Chart />
+    </div>
+    <router-link to="/top">
```

Comment:
```suggestion
    <router-link to="/top" tag="button">
```
とかの方がrouter-link使えてる感出ますね